### ISSUE TITLE
C Library

### ISSUE THREAD

**Issue by part1san** [sippy/rtpproxy#128]
Hello.
There are C implementation of RTP Proxy protocol?)
What do you recommend?

**Comment by sobomax** [comment]
Hi @part1san thanks for your interest! I wish there would be one, just never got around putting one around. :( AFAIK there is  @vasilakisfil working on Rust version, which might be the closest thing to that. There is also Python implementation, which might be used as a "reference":

https://github.com/sippy/b2bua/blob/master/sippy/Rtp_proxy_client.py
https://github.com/sippy/b2bua/blob/master/sippy/Rtp_proxy_session.py

**Comment by vasilakisfil** [comment]
Yes, we are working on a very nice & friendly Rust library for rtpproxy, but it will be open sourced later this year. Will ping once that happens.

**Comment by part1san** [comment]
Okey. Thank you.
